source/acquaman/AMTimedScanActionController.h

```diff
@@ -38,6 +38,7 @@ Q_DECLARE_METATYPE(AMScanActionControllerBasicFileWriter::FileWriterError)
 #define AMTIMEDSCANACTIONCONTROLLER_FILE_ALREADY_EXISTS 265003
 #define AMTIMEDSCANACTIONCONTROLLER_COULD_NOT_OPEN_FILE 265004
 #define AMTIMEDSCANACTIONCONTROLLER_UNKNOWN_FILE_ERROR 265005
+#define AMTIMEDSCANACTIONCONTROLLER_FAILED_TO_WRITE_FILE 265006
```

Comment:
All of them should be different.  You are correct.  That is likely just laziness in copy paste that was never remedied.